### DISCUSSION TITLE
Is a single-node keeper deployment ok for s3queue?

### DISCUSSION THREAD

**Discussion by ct1n** [ClickHouse/ClickHouse#67830] [Q&A] [answered]
We need to use keeper in order to use s3queues but we're ok with occasional downtime. Is there any downside to using a single node clickhouse deployment with keeper enabled for s3queues? We're going to use amazon gp2 volumes with 0.2% annual failure rate. If the volume fails we're ok with reprocessing the data.

**Comment by den-crane** [comment, marked as answer]
No downsides

**Reply by ct1n** [reply]
Thanks!
